test/test_linearizer_failures.py

```diff
@@ -154,10 +154,11 @@ def test_failure_21(self):
     opts = [Opt(op=OptOps.PADTO, axis=0, amt=32)]
     helper_test_lin(Linearizer(ast), opts, failed_platforms=[])
 
+  @unittest.skipIf(CI and Device.DEFAULT=="METAL", "works fine on intel CI METAL")
```

Comment:
i think ci is using real metal now

Reply:
It fails locally for me, but succeeds on CI

Reply:
I think it's fine to skip this in CI, it's numerical issue and CI might succeed because it emulates half. My point was that CI is not using intel

Reply:
Ahhh sorry I'll fix the comment